### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by JasperCraeghs**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+This script allows you to connect your test reports to your test cases via the `mlx.traceability`_ Sphinx extension.
+
+------------
+Installation
+------------
+
+.. code-block:: console
+
+    python3 -m pip install mlx.xunit2rst
```
Comment (Letme) [inline README.rst]: Herenits nice to use `pip3 install mlx.xunit2rst`

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+-----
+Usage
+-----
+
+Unit test report
+================
+
+.. code-block:: console
+
+    python3 -m mlx.xunit2rst -i utest_report.xml -o utest_report.rst
```
Comment (Letme) [inline README.rst]: Create entry point so we dont need this python3 infront

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    twine check dist/*
+    check-manifest {toxinidir}
+    flake8 mlx tests setup.py
+
+[testenv:test]
+deps=
+    {[testenv]deps}
+    nose
+commands=
+    nosetests
```
Comment (Letme) [inline tox.ini]: Since tests are run, generate coverage as well. And add a badge to readme...

Comment (JasperCraeghs) [reply tox.ini]: Running `coverage` 3271189f6cb582b387d844f0a41c5393ab22a53b
Added badges 65b47f60062797d1e791a22f6410c0693f7fb74d

File: README.rst (hunk 2 above)
Comment (JasperCraeghs) [reply README.rst]: Entry point existed already, so I just simplified this line 4762a5ab74e569cb4e01680795218edb6a8f4f2b

File: README.rst (hunk 1 above)
Comment (JasperCraeghs) [reply README.rst]: done in 4762a5ab74e569cb4e01680795218edb6a8f4f2b

**Pull request merged**